src/main/com/yetanalytics/datasim/input.clj

```diff
@@ -1,35 +1,44 @@
 (ns com.yetanalytics.datasim.input
   "Comprehensive specification of input"
   (:require [clojure.spec.alpha :as s]
+            [clojure.walk :as w]
             [com.yetanalytics.datasim.protocols :as p]
+            [com.yetanalytics.pan :as pan]
             [com.yetanalytics.pan.objects.profile :as ps]
             [com.yetanalytics.pan.objects.pattern :as pat]
-            [xapi-schema.spec :as xs]
             [com.yetanalytics.datasim.input.profile :as profile]
             [com.yetanalytics.datasim.input.personae :as personae]
             [com.yetanalytics.datasim.input.alignments :as alignments]
             [com.yetanalytics.datasim.input.parameters :as params]
             [com.yetanalytics.datasim.io :as dio]
             [com.yetanalytics.datasim.util.xapi :as xapiu]
-            [clojure.walk :as w]
-            [com.yetanalytics.datasim.util :as u]))
+            [com.yetanalytics.datasim.util.errors :as errs]))
+
+;; TODO: Ideally we want to use the top-level Pan API functions (which now
+;; support multiple profiles).
+;; We apply this kludgey patch simply to minimize breakage.
 
 (defn- profiles->pedges
   [profiles]
-  (let [[templates patterns]
-        (reduce (fn [[ts ps] {:keys [templates patterns]}]
-                  [(concat ts templates) (concat ps patterns)])
-                [[] []]
-                profiles)]
-    (pat/get-edges (pat/create-graph templates patterns))))
+  (let [combined-profile
+        (->> profiles
+             (reduce (fn [m {:keys [templates patterns]}]
+                       (-> m
+                           (update :templates concat templates)
+                           (update :patterns concat patterns)))
+                     {:templates []
+                      :patterns  []}))]
+    (pat/get-edges (pat/create-graph combined-profile))))
+
+;; ::profiles is unused, but needs to stay or else tests will break
```

Comment:
The only reason I'm not also deleting or modifying the tests is because I'm not sure if that would technically count as a breaking change.

Reply:
Decided to switch over tests and delete unused specs anyways: https://github.com/yetanalytics/datasim/pull/65/commits/13512a381338024242b20faa6804f6ffcb1b2017. Doing so after helped me uncover the bug https://github.com/yetanalytics/project-pan/pull/31 in Pan.